cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -424,6 +424,7 @@ features:
         pageTitle: CDAP | Connections | Google BigQuery | {connectionId}
         title: Select table
       DatabaseBrowser:
+        defaultErrorMessage: Reading {tableId} failed. Please retry again.
```

Comment:
should we say `Please try again` instead of `Please retry again`? `retry` and `again` seems redundant